keyboards/monsgeek/m1/config.h

```diff
@@ -0,0 +1,90 @@
+/* Copyright (C) 2022 jonylee@hfd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* Use 6 dynamic keymap layers */
+#define DYNAMIC_KEYMAP_LAYER_COUNT 6
+
+/* Force NKRO on boot up regardless of the setting saved in the EEPROM (uncomment to enable it) */
+#define FORCE_NKRO
+
+/* encoder */
+// #define ENCODER_RESOLUTION 4
+// #define ENCODER_MAP_KEY_DELAY 10
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+/* sets the number of milliseconds to pause after sending a wakeup packet. Disabled by default, you might want to set this to 200 (or higher) if the keyboard does not wake up properly after suspending.*/
+#define USB_SUSPEND_WAKEUP_DELAY 400
+
+/* SPI Config for spi flash*/
+#define SPI_DRIVER SPIDQ
+#define SPI_SCK_PIN B3
+#define SPI_MOSI_PIN B5
+#define SPI_MISO_PIN B4
+#define SPI_MOSI_PAL_MODE 5
+
+#define EXTERNAL_FLASH_SPI_SLAVE_SELECT_PIN C12
+#define WEAR_LEVELING_BACKING_SIZE (8 * 1024)
+
+/* I2C Config for LED Driver */
+#define DRIVER_COUNT 2
+#define DRIVER_ADDR_1 0b1110100
+#define DRIVER_ADDR_2 0b1110111
+#define I2C1_SDA_PIN B7
+#define I2C1_SCL_PIN B6
+#define I2C1_SCL_PAL_MODE 4
+#define I2C1_OPMODE OPMODE_I2C
```

Comment:
These appear to be the default settings, and none of theme need to be defined.
```suggestion
```

Reply:
It seems like  need to keep `#define I2C1_OPMODE OPMODE_I2C` otherwise the following error message will appear:

> Compiling: platforms/chibios/drivers/i2c_master.c                                                  platforms/chibios/drivers/i2c_master.c:101:5: error: 'I2C1_OPMODE' undeclared here (not in a function); did you mean 'I2C1_BASE'?
     I2C1_OPMODE,
     ^~~~~~~~~~~
     I2C1_BASE
 [ERRORS]

Reply:
Yes; apparently `i2c_master.c` defines the default value of `I2C1_OPMODE` only for the `USE_I2CV1` case, but `USE_I2CV1` is not defined for WB32 chips.